docs/datasets/index.rst

```diff
@@ -23,6 +23,53 @@ The functions have a naming pattern (following the `sklearn.datasets` lead):
           And there is a separate section describing the :ref:`dataformats`
           that are commonly used in gamma-ray astronomy.
 
+.. _gammapy-extra:
+
+gammapy-extra
+=============
+
+To keep the Gammapy code repository at https://github.com/gammapy/gammapy small and clean,
+we are putting sample data files and IPython notebooks in an extra repository
+at https://github.com/gammapy/gammapy-extra/ .
+
+To get the repository, ``git clone`` it to a location of your choosing using a git protocol of your choosing
+(try HTTPS or see the `Github clone URL help article`_ if you're not sure which you want).
+
+.. code-block:: bash
+
+    git clone https://github.com/gammapy/gammapy-extra.git
+    git clone git@github.com:gammapy/gammapy-extra.git
+
+If you don't have git, you can also fetch the latest version as a zip file:
+
+.. code-block:: bash
+
+    wget https://github.com/gammapy/gammapy-extra/archive/master.zip
+    unzip master.zip # will result in a `gammapy-extra-master` folder
+
+The Gammapy tests, docs generator, examples and tutorials will access files from the ``gammapy-extra``
+repo using the `gammapy.datasets.gammapy_extra` object.
+
+For this to work, you have to set the ``GAMMAPY_EXTRA`` shell environment variable to point to that folder.
+We suggest you put this in you ``.bashrc`` or ``.profile``
+
+    export GAMMAPY_EXTRA=/path/on/your/machine/to/gammapy-extra
+
+After you've done this, open up a new terminal (or ``source .profile``) and check if ``gammapy-extra`` is found:
+
```

Comment:
Is the `gammapy-extra` repo really checked out at `/home/jon/Pycharm/gammapy-extra/gammapy-extra`?

Maybe it's at ``/home/jon/Pycharm/gammapy-extra/`?

This is the check I put in:
https://github.com/gammapy/gammapy/blob/master/gammapy/datasets/core.py#L226

Do you have that file? At what location?